spec/models/fits_datastream_spec.rb

```diff
@@ -3,9 +3,10 @@
 describe FitsDatastream, type: :model, unless: $in_travis do
   describe "image" do
     before(:all) do
+      ActiveFedora::Cleaner.clean!
```

Comment:
@jcoyne Is this needed?  I thought this line would do that: https://github.com/projecthydra/sufia/blob/master/spec/spec_helper.rb#L112